sdk/tools/rsym/rsym.h

```diff
@@ -26,7 +26,7 @@ typedef struct {
   USHORT s_nlnno;    /* number of line number entries    */
   ULONG  s_flags;    /* flags                            */
 } SCNHDR;
-#pragma pack(4)
+#pragma pack(push, 4)
```

Comment:
Shouldn't the two structures above pragma-pack-push-4'ed too?

Reply:
Don't know, I simply fixed the bug that it was not properly restored....